test/datastore/e2e/utils.dart

```diff
@@ -20,7 +19,7 @@ const TEST_BLOB_INDEXED_PROPERTY = 'blobPropertyIndexed';
 final TEST_BLOB_INDEXED_VALUE = BlobValue([0xaa, 0xaa, 0xff, 0xff]);
 
 Key buildKey(int i,
-    {Function idFunction, String kind = TEST_KIND, Partition p}) {
+    {Function? idFunction, String kind = TEST_KIND, Partition? p}) {
```

Comment:
Could `idFunction` be more specific? At least `Object Function(Object)`? Or more specific?